dev-haskell/haskell-src-exts/haskell-src-exts-1.23.1.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# ebuild generated by hackport 0.6.6.9999
+#hackport: flags: +base4
+
+CABAL_FEATURES="lib profile haddock hoogle hscolour test-suite"
+inherit haskell-cabal
+
+DESCRIPTION="Manipulating Haskell source: abstract syntax, lexer, parser, and pretty-printer"
+HOMEPAGE="https://github.com/haskell-suite/haskell-src-exts"
+SRC_URI="https://hackage.haskell.org/package/${P}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0/${PV}"
```

Comment:
The slot is probably wrong. Current latest slot is `1.22`, not `0`.

Reply:
Hrm, the slot here came directly from hackport, and it seems to be consistent with other packages, including this one up to v1.20.3

I'm not sure why v1.21 and v1.22 moved the version from a sub-slot to a regular slot, but also I believe those two versions are currently masked so...should we just keep it here in the sub-slot?

Reply:
I think it was slotted to try to provide multiple library versions while we port all depends. Ideally we should eventually unslot it, but we are not ready yet to do it. I suggest keeping the current SLOT schema.

Reply:
No problem, I'll make the change.